docker-compose.yml

```diff
@@ -49,6 +49,7 @@ services:
       - RELAYER_REGISTRY_ADDRESSES=
       - RELAYER_ALLOW_TX_QUERIES=true
       - RELAYER_ALLOW_KV_CALLBACKS=true
+      - RELAYER_STORAGE_PATH=storage/leveldb
```

Comment:
let's stick to the pattern of folders that entities are use, e.g. home dir for neutron chain is `/data/test-1`, so let's make the relayer's storage folder be similar to it, for instance `/data/relayer/storage/leveldb`